providers/src/airflow/providers/apache/spark/hooks/spark_submit.py

```diff
@@ -518,9 +518,13 @@ def _build_track_driver_status_command(self) -> list[str]:
 
     def _resolve_kerberos_principal(self, principal: str | None) -> str:
         """Resolve kerberos principal."""
-        from airflow.security.kerberos import get_kerberos_principle
+        # todo: remove try/exception when min airflow version is 3.0
+        try:
+            from airflow.security.kerberos import get_kerberos_principal
+        except ModuleNotFoundError:
```

Comment:
```suggestion
        except ImportError:
```

I think this should be `ImportError`? You are trying to import a function not a module